eox_hooks/tests/test_tasks.py

```diff
@@ -0,0 +1,27 @@
+"""This file contains all the test for the tasks.py file.
+
+Classes:
+    SingalsTestCase: Test tasks.
+"""
+from django.test import TestCase
+from mock import patch
+
+from eox_hooks.tasks import default_task
+
+
+class TestTasks(TestCase):
+    """Tasks test class."""
+
+    def setUp(self):
+        self.kwargs = {
+            "course_id": "course_id",
+            "username": "username",
+        }
+        self.sender = "sender.example"
+
+    @patch('eox_hooks.tasks.log.info')
+    def test_default_task(self, log):
```

Comment:
it is better if you test the log with `LogCapture`

Reply:
Done! Also added checking the message